platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -71,6 +71,34 @@ Starting with Gradle 9.0, the Gradle API is annotated using JSpecify instead.
 
 For more details and potential breakages, see the dedicated [upgrading guide section](userguide/upgrading_version_8.html).
 
+### Task graph diagnostic
+
+We introduce a new task graph diagnostic feature to help you understand the task graph of your build.
+You can use `--task-graph` command line option to print the task graph of your build without executing the tasks, for example:
+```
+$ ./gradlew root r2 --task-graph
+
```

Comment:
Please remove the `//` below, the goal is to seperate the example into two code snippets (instead of one):

```suggestion
//```
This prints a visual representation of the task graph for the specified tasks:
//```
```

Reply:
Lifehack for the future: you can use more than three ticks to have nested code blocks. Like this:
`````
````suggestion
some words
```kotlin
some code
```
````
`````